packages/app-project/stores/YourStats.js

```diff
@@ -0,0 +1,90 @@
+import { autorun } from 'mobx'
+import { addDisposer, flow, getRoot, types } from 'mobx-state-tree'
+import auth from 'panoptes-client/lib/auth'
+import asyncStates from '@zooniverse/async-states'
+import { panoptes } from '@zooniverse/panoptes-js'
+
+const YourStats = types
+  .model('YourStats', {
+    dailyCounts: types.optional(types.array(types.frozen({})), []),
+    error: types.maybeNull(types.frozen({})),
+    loadingState: types.optional(types.enumeration('state', asyncStates.values), asyncStates.initialized),
+    totalCount: types.optional(types.number, 0)
+  })
+
+  .views(self => ({
+    get todaysCount () {
+      return self.dailyCounts[0]
+    }
+  }))
+
+  .actions(self => {
+    function createProjectObserver () {
+      const projectDisposer = autorun(() => {
+        const { project, user } = getRoot(self)
+        if (project.id && user.id) {
+          self.fetchActivityCount()
+          self.fetchDailyCounts()
+        }
+      })
+      addDisposer(self, projectDisposer)
+    }
+
+    return {
+      afterAttach () {
+        createProjectObserver()
+      },
+
+      fetchActivityCount: flow(function * fetchActivityCount () {
+        const { project, user } = getRoot(self)
+        self.loadingState = asyncStates.loading
+        try {
+          const token = yield auth.checkBearerToken()
+          const authorization = `Bearer ${token}`
+          const query = {
+            project_id: project.id,
+            user_id: user.id
+          }
+          const response = yield panoptes.get('/project_preferences', query, authorization)
+          const [ preferences ] = response.body.project_preferences
+          self.totalCount = preferences ? preferences.activity_count : 0
+        } catch (error) {
+          console.log(error.message)
```

Comment:
Maybe we should log the full error so we can get the error code and any other relevant info?

Reply:
Good point. I took it out to get rid of noise in the tests.

Reply:
I'm going to log the full error, but suppress the console during tests.